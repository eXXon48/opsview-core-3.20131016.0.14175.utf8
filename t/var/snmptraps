192.168.10.20
192.168.10.20
SNMPv2-MIB::sysUpTime.0 4:20:49:47.73
SNMPv2-MIB::snmpTrapOID.0 SNMPv2-SMI::enterprises.9.9.43.2.0.1
SNMPv2-SMI::enterprises.9.9.43.1.1.6.1.3.45 1
SNMPv2-SMI::enterprises.9.9.43.1.1.6.1.4.45 2
SNMPv2-SMI::enterprises.9.9.43.1.1.6.1.5.45 3
SNMP-COMMUNITY-MIB::snmpTrapAddress.0 192.168.10.20
SNMP-COMMUNITY-MIB::snmpTrapCommunity.0 "public"#---next trap---#
192.168.10.20
192.168.10.20
SNMPv2-MIB::sysUpTime.0 0:0:00:22.66
SNMPv2-MIB::snmpTrapOID.0 SNMPv2-MIB::coldStart
SNMPv2-MIB::sysUpTime.0 0:0:00:22.66
SNMPv2-SMI::enterprises.9.2.1.2.0 "reload"
SNMP-COMMUNITY-MIB::snmpTrapAddress.0 172.16.0.1
SNMP-COMMUNITY-MIB::snmpTrapCommunity.0 "public"
SNMPv2-MIB::snmpTrapEnterprise.0 SNMPv2-SMI::enterprises.9.1.186#---next trap---#
192.168.10.21
192.168.10.21
SNMPv2-MIB::sysUpTime.0 0:0:00:22.66
SNMPv2-MIB::snmpTrapOID.0 SNMPv2-MIB::coldStart
SNMPv2-MIB::sysUpTime.0 0:0:00:22.66
SNMPv2-SMI::enterprises.9.2.1.2.0 "reload"
SNMP-COMMUNITY-MIB::snmpTrapAddress.0 192.168.10.21
SNMP-COMMUNITY-MIB::snmpTrapCommunity.0 "public"
SNMPv2-MIB::snmpTrapEnterprise.0 SNMPv2-SMI::enterprises.9.1.186#---next trap---#
192.168.10.20
192.168.10.20
SNMPv2-MIB::sysUpTime.0 0:0:00:22.66
SNMPv2-MIB::snmpTrapMalformedOID.0 Malformed#---next trap---#
192.168.100.100
192.168.100.100
SNMPv2-MIB::sysUpTime.0 0:0:00:22.66
SNMPv2-MIB::snmpTrapOID.0 SNMPv2-MIB::coldStart
SNMPv2-MIB::sysUpTime.0 0:0:00:22.66
SNMPv2-SMI::enterprises.9.2.1.2.0 "reload"
SOLARWINDS-TRAPS::nodeIP 192.168.10.20
SNMP-COMMUNITY-MIB::snmpTrapAddress.0 172.16.0.1
SNMP-COMMUNITY-MIB::snmpTrapCommunity.0 "solarwindsforwarding"
SNMPv2-MIB::snmpTrapEnterprise.0 SNMPv2-SMI::enterprises.9.1.186#---next trap---#
