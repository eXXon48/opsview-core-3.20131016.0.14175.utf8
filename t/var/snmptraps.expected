$rulesets = {
              '81' => {
                        'rules' => [
                                     {
                                       'alertlevel' => '0',
                                       'message' => 'OK',
                                       'process' => '1',
                                       'code' => 'This will not compile'
                                     }
                                   ],
                        'servicename' => 'Test exceptions'
                      },
              '88' => {
                        'rules' => [],
                        'servicename' => 'snmptrap no freshness'
                      },
              '89' => {
                        'rules' => [],
                        'servicename' => 'snmptrap renotify'
                      },
              '79' => {
                        'rules' => [
                                     {
                                       'alertlevel' => '1',
                                       'message' => 'Device coldstarted',
                                       'process' => '1',
                                       'code' => '"${TRAPNAME}" =~ /SNMPv2-MIB::coldstart/i'
                                     },
                                     {
                                       'alertlevel' => '0',
                                       'message' => 'OK',
                                       'process' => '1',
                                       'code' => '1'
                                     }
                                   ],
                        'servicename' => 'Coldstart'
                      },
              '82' => {
                        'rules' => [
                                     {
                                       'alertlevel' => '0',
                                       'message' => 'OK',
                                       'process' => '1',
                                       'code' => '0'
                                     }
                                   ],
                        'servicename' => 'Another exception'
                      }
            };
$advsnmptrap = {
                 '192.168.10.23' => [
                                    '79',
                                    '81',
                                    '82',
                                    '88',
                                    '89'
                                  ],
                 '192.168.10.20' => [
                                    '79',
                                    '81',
                                    '82'
                                  ],
                 '67.207.142.209' => [
                                       '79',
                                       '81',
                                       '82',
                                       '88',
                                       '89'
                                     ],
                 '10.20.20.20' => [
                                    '79',
                                    '81',
                                    '82',
                                    '88',
                                    '89'
                                  ],
                 '192.168.10.22' => [
                                    '79',
                                    '81',
                                    '82',
                                    '88',
                                    '89'
                                  ]
               };
$hostip_lookup = {
                   '192.168.10.23' => [
                                      'cisco1'
                                    ],
                   '192.168.10.20' => [
                                      'cisco'
                                    ],
                   '67.207.142.209' => [
                                         'cisco1'
                                       ],
                   '10.20.20.20' => [
                                      'cisco1'
                                    ],
                   '192.168.10.22' => [
                                      'cisco2',
                                      'cisco3'
                                    ]
                 };
$trace = {
           'cisco2' => 1,
           'cisco1' => 1
         };
$debug = undef;
$enabled = 1;
1;
